packages/mira/tasks/amr_to_mmt.py

```diff
@@ -12,23 +13,44 @@ def main():
     exitCode = 0
 
     try:
+        start = time.time()
         taskrunner = TaskRunnerInterface(description="AMR to MMT")
         taskrunner.on_cancellation(cleanup)
 
         data = taskrunner.read_input_str_with_timeout()
         amr = json.loads(data)
+
+        # Remove any empty mathML expressions
+        if 'semantics' in amr and "ode" in amr["semantics"]:
+            ode = amr["semantics"]["ode"]
+            if "parameters" in ode:
+                parameters = ode["parameters"]
+                for param in parameters:
+                    if "units" in param and param["units"] and "expression" in param["units"]:
+                        unit_expression = param["units"]["expression"]
+                        if unit_expression == None or unit_expression == "":
+                            del param["units"]
+
         mmt = model_from_json(amr)
 
+        end = time.time()
+        taskrunner.log(f"model from json time since {(end - start) * 1000} ms")
+
         template_params = {}
 
         count = 0
+
+        taskrunner.log("")
+        taskrunner.log(f"number of templates {len(mmt.templates)}");
+
         for tm in mmt.templates:
             # Sanitize
             count = count + 1
             if tm.name == None or tm.name == "":
                 tm.name = "generated-" + str(count)
 
-            params = tm.get_parameter_names()
+            # params = tm.get_parameter_names()
```

Comment:
I am not following why we are no longer grabbing param names here?
It looks like our `entry` below will always have an empty list?

Reply:
was from perf test a while ago, will revert, thx